views/home.html

```diff
@@ -214,19 +214,35 @@ <h3 class="f2-light pt-4">{{{localized.need_help.title}}}</h3>
 
   <div class='py-7 py-md-8 py-lg-9' id='apps'>
     <div class="container-lg text-center">
-      <h1 class="f0-light">{{{localized.apps.title}}}</h1>
+      <h1 class="f0-light">{{{localized.companies.title}}}</h1>
       <p class="lead">
-        {{{localized.apps.description}}}
+        {{{localized.companies.description}}}
       </p>
 
-      <hr>
+      <div class="my-4">
+        <div class="text-center">
+          <ul class="d-flex flex-justify-around flex-wrap list-style-none my-2 my-md-3">
+            {{#each companies}}
+              <li class="featured-company-list-item">
+                <a href="{{ this.url }}" target="_blank">
```

Comment:
[Links to cross-origin destinations are unsafe](https://web.dev/external-anchors-use-rel-noopener/)
```suggestion
                <a href="{{ this.url }}" rel="noopener" target="_blank">
```